### PR TITLE
Minor: Remove the warning when an event apart from To-Device is received

### LINKED ISSUE #61: Figure out what the incoming non to-device events are

### PR DESCRIPTION
**Pull request by daniel-abramov**
Fixes https://github.com/matrix-org/waterfall/issues/61

### CODE REVIEW

**Review by SimonBrandner — CHANGES_REQUESTED** [review]
Let's figure out what these are before we remove the warning: https://github.com/matrix-org/waterfall/issues/61

File: pkg/signaling/client.go [hunk 1]
```diff
@@ -45,7 +45,6 @@ func (m *MatrixClient) RunSyncing(callback func(*event.Event)) {
 	syncer.OnEvent(func(_ mautrix.EventSource, evt *event.Event) {
 		// We only care about to-device events.
```
Comment (SimonBrandner) [inline pkg/signaling/client.go:46]:
```suggestion
		// We only care about to-device events but also receive m.presence and
		// m.push_rules events; we can simply ignore those.
```

**Review by SimonBrandner — CHANGES_REQUESTED** [review]
LGTM after the added comment

**Comment by daniel-abramov** [comment]
> Let's figure out what these are before we remove the warning: #61

You seem to have figured it out already, right? :)

OT: Btw, the presence events don't seem to be used by Element Web and Element Desktop, right?

**Comment by daniel-abramov** [comment]
> LGTM after the added comment

Done!

**Review by SimonBrandner — APPROVED** [review]

**Pull request merged**